src/breadcrumbs/stories/Breadcrumbs.stories.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { Meta } from "@storybook/react";
+
+import "./index.css";
+import { Breadcrumbs } from "../Breadcrumbs";
+import { BreadcrumbLink } from "../BreadcrumbLink";
+
+export default {
+  title: "Component/Breadcrumb",
+} as Meta;
+
+export const Default = () => {
+  return (
+    <Breadcrumbs className="breadcrumb">
+      <ol>
+        <li>
+          <BreadcrumbLink href="https://www.w3.org/TR/wai-aria-practices-1.1/">
+            WAI-ARIA Authoring Practices 1.1
+          </BreadcrumbLink>
+        </li>
+        <li>
```

Comment:
I think the API should look like this, because users should not care about the markup nor how to use regular breadcrumbs in semantic html manner.

```js
<Breadcrumb>
  <BreadcrumbItem>
    <BreadcrumbLink href="#">Home</BreadcrumbLink>
  </BreadcrumbItem>

  <BreadcrumbItem>
    <BreadcrumbLink href="#">Docs</BreadcrumbLink>
  </BreadcrumbItem>

  <BreadcrumbItem isCurrentPage>
    <BreadcrumbLink href="#">Breadcrumb</BreadcrumbLink>
  </BreadcrumbItem>
</Breadcrumb>
```

Reply:
I accepts what you are saying but,
How can we do this.

Primary rule of Reakit.
>  Each component returns a single HTML element

Take Radio as example,
https://reakit.io/docs/radio/

It's the users responsibility to provide a `label` around `Radio`
![image](https://user-images.githubusercontent.com/39694575/92473343-331ddd00-f1f8-11ea-906f-4d4a7eff54ea.png)

Reply:
Okay we can keep this, and approach the more declarative api when we will be building the designed components.

Reply:
Yup, we can remove the other parts when building a UI component.